clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -2846,6 +2862,15 @@ void CodeGenModule::AddGlobalAnnotations(const ValueDecl *D,
     Annotations.push_back(EmitAnnotateAttr(GV, I, D->getLocation()));
 }
 
+// Add "sycl-unique-id" llvm IR attribute for global variables marked with
+// SYCL device_global attribute, and return a unique string using
+// __builtin_sycl_unique_stable_id.
```

Comment:
The following function doesn't return anything.
```suggestion
// Add "sycl-unique-id" llvm IR attribute that contains a unique string
// generated by __builtin_sycl_unique_stable_id for global variables marked with
// SYCL device_global attribute.
```